util/gas.ts

```diff
@@ -732,3 +932,24 @@ export const findMatchingForkName = (
 
   return foundFork
 }
+
+const getCommonParam = (
+  common: Common | EOFCommon,
+  topic: string,
+  name: string,
+): bigint => {
+  if (common instanceof Common) {
+    return common.param(topic, name)
+  } else {
+    let formattedName = name
+    if (!name.endsWith('Gas')) {
+      formattedName = name
+        .replace('Cost', '')
+        .replace('Price', '')
+        .replace('Gas', '')
+        .concat('Gas')
+    }
+    // @ts-ignore it's warn on ide, but technically it's work
```

Comment:
it is warning because I try to install two different built (once on local submodule, once on remote registry), but I did link to different package names as in `setup_monorepo.sh`

Reply:
the name's EIP param set has been changed on the EOF-supported version in `ethjs`: https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/evm/src/params.ts.

I also have been asked in their community, there is no built-in function to transform from pre-EOF and EOF-supported version

Reply:
@charisra could you confirm whether the answer satisfied your question or not?